packages/smooth_app/lib/helpers/picture_capture_helper.dart

```diff
@@ -1,39 +1,105 @@
+import 'dart:io';
+import 'dart:math';
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:openfoodfacts/openfoodfacts.dart';
 import 'package:provider/provider.dart';
 import 'package:smooth_app/data_models/continuous_scan_model.dart';
 import 'package:smooth_app/database/product_query.dart';
-import 'package:smooth_app/generic_lib/loading_dialog.dart';
+import 'package:workmanager/workmanager.dart';
+
+@pragma(
+    'vm:entry-point') // Mandatory if the App is obfuscated or using Flutter 3.1+
+void callbackDispatcher() {
+  Workmanager().executeTask(
+    (String task, Map<String, dynamic>? inputData) async {
```

Comment:
This Map should be serialized/deserialized in a class

Reply:
crreated a seperate class in another file for it , I hope that's okay

Reply:
Have you fully tested this code and ensured that the access to the DB works well?

Reply:
>Have you fully tested this code and ensured that the access to the DB works well?

Does it mean if i able to update to the server ?  Cuz i am not updating the local database for now and the server query does run and it's working fine

Reply:
No, I just mean: are you able to update the DB from the WorkManager, because I guess you have to make some specific code for that

Reply:
Currently i am **`just pushing the image`** to the server and not fetching or saving anything 

for now user has to manually refresh the product to fetch the details from the server
Was having troubles updating the DB from workmanager, seems like little tricky in that part(probably native android)